tasks/13.txt

```diff
@@ -1,3 +1,26 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+
+db.products.aggregate([
+    {
+        $group: {
+            _id:1,
+            "TotalOfCurrentProducts": { 
+                "$sum": { 
+                    "$cond": [{ "$eq": [ "$Discontinued", 0 ] }, 1 , 0 ]
+                }
+            },
+            "TotalOfDiscontinuedProducts": { 
+                "$sum": { 
+                    "$cond": [{ "$eq": [ "$Discontinued", 1 ] }, 1, 0 ]
+                }
+            }
+        }
+    },
+    {
+        $project: {
+            _id:0
+        }
+    }
+])
```

Comment:
I think just 2 count functions will be better in the case. We can't use any index for the calculation here